src/Avalonia.Controls/Presenters/TextPresenter.cs

```diff
@@ -61,11 +92,104 @@ static TextPresenter()
 
         public TextPresenter()
         {
-            _caretTimer = new DispatcherTimer();
-            _caretTimer.Interval = TimeSpan.FromMilliseconds(500);
+            _text = string.Empty;
+            _caretTimer = new DispatcherTimer { Interval = TimeSpan.FromMilliseconds(500) };
```

Comment:
Not directly related to this PR: We already have an issue for spawning dedicated timers for handling undo in `TextBox` and this makes me wonder if we also really need a dedicated timer to just blink the caret. Makes it fairly expensive to construct these controls.

Reply:
Yeah, lets get this in first and then we can think about optimizing.

Reply:
New implementation will have a dedicated control for displaying a caret. The shape of the caret depends on the current font etc. So it makes sense to separate this logic.